resources/integration_test/tests/functions.yml

```diff
@@ -48,3 +48,28 @@ tests:
     expected_precision: [ 0, 10, 10, 0, 0, 0, 10, 10, 10, 10, 10, 10, 0, 10, 10, 10, 10, 10, 10 ]
     expected_scale: [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ]
     expected_nullability: [ 0, 0, 1, 1, 1, 1, 0, 0, 0, 1, 0, 1, 1, 1, 1, 0, 1, 1, 1 ]
+  - description: SQLGetTypeInfo function getall types
```

Comment:
Very cool, you were able to replace the GetTypeInfo unit tests with an entry here.